OneCommunityGlobal/HGNRest
# Fixing "Send Emails" visibility for Volunteers with 'Other Links' access and adding a new Send Emails Permission

# Description
![image](https://github.com/user-attachments/assets/92a9b4d4-eba4-4b53-b904-036bb8929fa9)
![image](https://github.com/user-attachments/assets/178d07f1-6e43-40d8-8b2a-f5c09aea21f0)

Controller method includes role and permission checks before sending emails in the backend.

## Related PRS (if any):
To test this backend PR you need to checkout the #2572 frontend PR.
…

## Main changes explained:
-Added checks to the controller method before sending out emails.
…

## How to test:
1. check into current branch
2. do npm install and ... to run this PR locally
3. Clear site data/cache
4. Please follow the remaining steps in frontend PR

## Screenshots or videos of changes:

## Note:
